### ISSUE TITLE
React Native

### ISSUE THREAD

**Issue by beeirl** [cuining/react-granim#2]
Does this work with React Native?

**Comment by cuining** [comment]
@Druux  It work with React. React Native maybe need special tags, such as `View` and `Text`.

**Comment by beeirl** [comment]
@cuining yes of course. are there any chances that you could provide a version for react native? or would you accept a pull request?

**Comment by cuining** [comment]
@Druux if you are willing to do this, i'm super glad to take it 😄
